src/contracts/pods/EigenPod.sol

```diff
@@ -434,26 +434,25 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     /// @notice Called by the EigenPodManager to fulfill a partial withdrawal proof request
     function fulfillPartialWithdrawalProofRequest(
-        IEigenPodManager.Journal calldata journal,
+        IEigenPod.VerifiedPartialWithdrawal calldata verifiedPartialWithdrawal,
         uint64 feeGwei,
         address feeRecipient
     ) external onlyEigenPodManager onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_WITHDRAWAL) {
 
-        require(journal.mostRecentWithdrawalTimestamp == mostRecentWithdrawalTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: proven mostRecentWithdrawalTimestamp must match mostRecentWithdrawalTimestamp in the EigenPod");
-        require(mostRecentWithdrawalTimestamp < journal.endTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: mostRecentWithdrawalTimestamp must precede endTimestamp");
+        require(verifiedPartialWithdrawal.mostRecentWithdrawalTimestamp == mostRecentWithdrawalTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: proven mostRecentWithdrawalTimestamp must match mostRecentWithdrawalTimestamp in the EigenPod");
+        require(mostRecentWithdrawalTimestamp < verifiedPartialWithdrawal.endTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: mostRecentWithdrawalTimestamp must precede endTimestamp");
 
-
-        uint256 provenPartialWithdrawalSumGwei = journal.provenPartialWithdrawalSumGwei;
-        require(sumOfPartialWithdrawalsClaimedGwei <= provenPartialWithdrawalSumGwei - feeGwei, "EigenPod.fulfillPartialWithdrawalProofRequest: sumOfPartialWithdrawalsClaimedGwei must be less than or equal to provenPartialWithdrawalSumGwei + feeGwei");
+        require(sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei + feeGwei <= verifiedPartialWithdrawal.provenPartialWithdrawalSumGwei, "EigenPod.fulfillPartialWithdrawalProofRequest: proven sum must be less than or equal to provenPartialWithdrawalSumGwei + feeGwei");
+        uint64 provenPartialWithdrawalSumGwei = verifiedPartialWithdrawal.provenPartialWithdrawalSumGwei;
         provenPartialWithdrawalSumGwei -= feeGwei;
 
 
         // subtract an partial withdrawals that may have been claimed via merkle proofs
-        provenPartialWithdrawalSumGwei -= sumOfPartialWithdrawalsClaimedGwei;
-        sumOfPartialWithdrawalsClaimedGwei = 0;
+        provenPartialWithdrawalSumGwei -= sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei;
+        sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei = 0;
```

Comment:
we should only be doing this logic if `sumOfPartialWithdrawalsClaimedViaMerkleProvenGwei != 0`

Reply:
ah I removed that logic from before for some reason smh, added back